boilerplate/openshift/golang_osd_cluster_operator/standard.mk

```diff
@@ -71,8 +71,8 @@ docker-push: push
 
 .PHONY: gocheck
 gocheck: ## Lint code
-	boilerplate/openshift/golang_osd_cluster_operator/golint.sh
-	${GOENV} go vet ./cmd/... ./pkg/...
+	boilerplate/_lib/ensure.sh golangci-lint
+	GOLANGCI_LINT_CACHE=/tmp/golangci-cache golangci-lint run -c boilerplate/openshift/golang_osd_cluster_operator/golangci.yml ./...
```

Comment:
Could put a comment here explaining why we need `GOLANGCI_LINT_CACHE`, pointing to that issue.